src/state/image-dataset/firebase/index.ts

```diff
@@ -90,7 +90,7 @@ class FirebaseRequest implements ImageDataset {
             });
     };
 
-    public setCollectionRef = (id: string) => {
+    private setCollectionRef = (id: string) => {
```

Comment:
the only substantive change in this file is to change this from public to private.  The rest of the file just got auto-formatted.